### PR TITLE
Added tempo changing

### PR DESCRIPTION
**Pull request by Raven13111**

### CODE REVIEW

File: playSong.py [hunk 1]
```diff
@@ -188,12 +188,24 @@ def parseInfo():
 
 	return notes
 
+def adjustTempoForCurrentNote():
+    global playback_speed, infoTuple, storedIndex
+    tempo_changes = infoTuple[3]
```
Comment (atirut-w) [inline playSong.py:193]: Are you sure `infoTuple` has enough information here? It's causing an `IndexError`.

See this file for testing, it has lots of tempo changes: [The Legend of Zelda, Ocarina of Time - Horse Race Theme.zip](https://github.com/Stereo101/python-MidiToVirtualPianoMacro/files/14549542/The.Legend.of.Zelda.Ocarina.of.Time.-.Horse.Race.Theme.zip)

**Comment by atirut-w** [comment]
Please be sure to create a feature branch next time you make a PR, unless you want to see merge or local-remote conflicts when someone commit to the master branch while your PR is not merged :+1:

File: playSong.py (hunk 1 above)
Comment (Raven13111) [reply playSong.py:193]: I gave wrong version

**Review by noigamegun — COMMENTED** [review]
As an end user I don’t really see a point in this. And what is the feature trying to do? **Speed** is just an offset of **pitch** and **tempo** but since it a **macro**, **pitch** doesn’t matter so it just leave us with **tempo**. And the program already have speed control.

**Comment by Raven13111** [comment]
> As an end user I don’t really see a point in this. And what is the feature trying to do? **Speed** is just an offset of **pitch** and **tempo** but since it a **macro**, **pitch** doesn’t matter so it just leave us with **tempo**. And the program already have speed control.

Some midis like rush e have it so it's changing constantly, this makes it sound fine or atleast better

**Comment by Raven13111** [comment]
Closing pr

**Pull request closed without merging**